### PR TITLE
Activeadmin

### PR DESCRIPTION
**Pull request by andrestobelem**
# Summary
* I added support for activeadmin.
* You can login with admin@example.com and password in /admin.
* You can create articles and comments thought the admin.

# Trello Card

https://trello.com/c/AyugnvwK

### CODE REVIEW

File: spec/models/admin_user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```
Comment (epintos) [inline spec/models/admin_user_spec.rb]: Try not to leave unused code. Its a good practice to make your own code review when you create a Pull request so you can detect this little things or other ones more severe

File: config/.secrets.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+mandril:
```
Comment (epintos) [inline config/.secrets.yml]: If you are still using app configuration, this file should be ignored

Comment (andrestobelem) [inline config/.secrets.yml]: My bad, i did something wrong with git.

Comment (andrestobelem) [inline config/.secrets.yml]: And I moved to the root of the app.

**Comment by epintos** [comment]
:+1:

**Pull request closed without merging**